### DISCUSSION TITLE
YAML format of file

### DISCUSSION THREAD

**Discussion by epajfl00** [make-all/tuya-local#728] [Q&A] [answered]
Hopefully a simple question to have answered.
Why does the entity format differ between the primary and secondary part?
One has a dash (-) and the other does not.
Seem inconsistent.

primary_entity:
  entity: cover

secondary_entities:
  - entity: switch

**Comment by make-all** [comment, marked as answer]
primary_entity specifies a single entity.  secondary_entities is a list of entities.

**Reply by epajfl00** [reply]
HI Jason,
Seems that you misunderstood my question, I was referring to the syntax, entity under Primary does not have "-" dash, while the entity under Secondary does have a dash, so it seems inconsistent. Is there a reason? 
I do understand the Primary and Secondary parts.

![Tuyalocal entity question 2](https://github.com/make-all/tuya-local/assets/86779948/4e637ef1-37ce-485a-a90e-d1f4da564d77)

**Reply by make-all** [reply]
No, I think you misunderstood the answer.

**Comment by epajfl00** [comment]
OK, in that case I must have misunderstood the answer.
So in this case, it must mean the following;
Yes it was intentional, to distinguish the Primary entity from the Secondary entities.
I now understand, thanks.

**Reply by make-all** [reply]
It means that in YAML, a line starting with a `- ` is the start of a list item.

**Comment by epajfl00** [comment]
Actually, that makes perfect sense now, excellent, many thanks for clarifying this basic YAML information, which I had misunderstood.
Sorry for wasting your time.
